samples/cellular/smp_svr/README.rst

```diff
@@ -0,0 +1,18 @@
+.. _smp_svr:
+
```

Comment:
I'd say you do not need to have the sample description as a separate document, as there are no other files to be added (at least for now). Meaning, you can have the description documentation in this rst file. Also, please take a look at the sample template https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html and follow is as far as possible.